plugins/builder/osbuild.py

```diff
@@ -31,6 +31,10 @@
 from koji.tasks import BaseTaskHandler
 
 
+DEFAULT_COMPOSER_URL = "http://localhost:8701/"
+DEFAULT_KOJIHUB_URL = "https://localhost/kojihub"
```

Comment:
Why are these the defaults? Or why are there defaults at all? Maybe we should make the composer one image-builder.osbuild.org? I think Lars wanted to change the port to 443 before the release.

Reply:
First of all easy testing: In the unit test I use those with `httpretty` to simulate with those to mock a composer conversation. Could also be done via always having a config, but that was easier.

Reply:
But yes, can totally change that to "image-builder.osbuild.org".